### ISSUE TITLE
How to run as a scheduled task with –OutputPath parameter ?

### ISSUE THREAD

**Issue by jheregknt** [alanrenouf/vCheck-vSphere#438]
Is there a way to run vcheck as a scheduled task  with the –OutputPath parameter? 

We are trying to just create a file which we can then upload to a website  in a fully automated process.

**Comment by Sneddo** [comment]
Of course, same as any other scheduled task
Command: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
Arguments: -Command "& 'C:\Scripts\Powershell\vCheck\vcheck.ps1' -Outputpath C:\Scripts\Powershell\vCheck"

Adjust paths as required.

**Comment by jheregknt** [comment]
Unfortunately after adjusting the paths and putting the  -Outputpath in the
correct spot it still does not work .

It runs for a second then gives a Last Run Result of (0x1)

Windows Server 2012R2

Program/script: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe

Add Arguments (optional): -Command "& 'C:\Scripts\vcheck.ps1' -Outputpath
C:\VcheckOutput"


Any help would be greatly appreciated!

**Comment by Sneddo** [comment]
I guess a couple of things to try:
- Make sure the task is set to run whether the user is logged on ornot
- The user has logon as a batch rights
- Set the working directory to C:\Scripts\

**Comment by jheregknt** [comment]
I have confirmed those 3 things and it still did not work.

Strangely I got it to work two different ways in Task Scheduler for 2012 R2
  first use PowerCLI 5.5, not 6.

1. Create task as a Basic Task and put the entire command in the
Program/Script line:
 (C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -Command "&
'C:\Scripts\vCheck-vSphere-master\vcheck.ps1' -Outputpath C:\VcheckOutput")
 and Windows will identify the argument, remove it and add it to the Add
argument line , just add

2. Create a batch file with just the entire command line , and place the
batch file  in the Program/Script line
(C:\Scripts\vCheck-vSphere-master\vcheck.bat), make sure to point it to the
batch file in the Start in (optional) text box

Not sure why but it's working now !

Thanks again!
